Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM quay.io/ukhomeofficedigital/docker-centos-base
 
+ENV NODE_VERSION v4.2.2
```

Comment:
Maybe move this line right before you actually need the variable? That'd save us from re-creating layers for things like `WORKDIR /opt/nodejs` which don't change.